samples/nrf9160/modem_shell/src/th/th_ctrl.c

```diff
@@ -112,32 +113,47 @@ static void th_ctrl_work_handler(struct k_work *work_item)
 #if defined(CONFIG_MOSH_IPERF3)
 		if (!strcmp(data->argv[0], "iperf3")) {
 			ret = iperf_main(data->argc, data->argv, data->results_str,
-					TH_RESPONSE_BUFFER_SIZE, &(data->kill_signal));
+					 TH_RESPONSE_BUFFER_SIZE, &(data->kill_signal));
 		}
 #endif
 #if defined(CONFIG_MOSH_PING)
 		if (!strcmp(data->argv[0], "ping")) {
 			ret = icmp_ping_shell_th(data->shell, data->argc, data->argv,
-					      data->results_str,
-					      TH_RESPONSE_BUFFER_SIZE,
-					      &(data->kill_signal));
+						 data->results_str,
+						 TH_RESPONSE_BUFFER_SIZE,
+						 &(data->kill_signal));
 		}
 #endif
 	} else {
 		assert(data->results_str == NULL);
 
+		if (data->pipeline) {
+			for (int cmd_ix = 0; cmd_ix < data->argc; cmd_ix++) {
+				mosh_print("Executing command %d \"%s\"...",
+				cmd_ix, data->argv[cmd_ix]);
```

Comment:
Check that the indentation is not at same level with `mosh_print()`. These are sometimes wrong in github PR view so not sure.